package.json

```diff
@@ -39,6 +39,8 @@
 		"@babel/preset-react": "^7.12.13",
 		"@babel/runtime": "7.14.0",
 		"@material-ui/icons": "^4.11.2",
+		"@radix-ui/react-radio-group": "^0.1.0",
```

Comment:
Minor note - we should always pin dependencies to a specific version, rather than use ranges. We need to set up renovatebot for this repo ASAP.

Reply:
(this can be done with `npm i -S <package>`)